artifacts/definitions.py

```diff
@@ -34,9 +34,8 @@
     'Cloud Storage': 'Cloud storage artifacts.',
     'Configuration Files': 'Configuration files artifacts.',
     'Execution': 'Contain execution events.',
-    'ExternalAccount': (
-        'Information about any users\' account, e.g. username, '
-        'account ID, etc.'),
+    'ExternalAccount': ('Information about any user accounts e.g. username, '
+                        'account ID, etc.'),
     'External Media': 'Contain external media data or events e.g. USB drives.',
     'KnowledgeBase': 'Artifacts used in knowledge base generation.',
```

Comment:
Please remove duplicate key: KnowledgeBase

************* Module artifacts.definitions
W: 29, 9: Duplicate key 'KnowledgeBase' in dictionary (duplicate-key)